vendor/golang.org/x/sys/unix/file_unix.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package unix
+
+import (
+	"os"
+	"syscall"
+)
+
+// FIXME: unexported function from os
+// syscallMode returns the syscall-specific mode bits from Go's portable mode bits.
+func syscallMode(i os.FileMode) (o uint32) {
```

Comment:
__[gometalinter]__
* [deadcode] syscallMode is unused

<!-- Comment by [SideCI](https://sideci.com) -->